astropy/table/tests/test_table.py

```diff
@@ -1445,13 +1498,13 @@ def test_equality_masked():
     t = table.Table(t, masked=True)
 
     # All rows are equal
-    assert np.all(t == t)
+    assert np.all(t.rows_equal(t))
 
     # Assert no rows are different
     assert not np.any(t != t)
 
     # Check equality result for a given row
-    assert np.all((t == t[3]) == np.array([0, 0, 0, 1, 0, 0, 0, 0], dtype=bool))
+    assert np.all((t.rows_equal(t[3])) == np.array([0, 0, 0, 1, 0, 0, 0, 0], dtype=bool))
```

Comment:
This test is passing when I execute it 
```
np.all((t.rows_equal(t[3])) == np.array([0, 0, 0, 1, 0, 0, 0, 0], dtype=bool))
Out[3]: True
```
But it keeps failling when running test it keeps failing I have no idea why it is giving this error
```
E       DeprecationWarning: elementwise == comparison failed; this will raise an error in the future.
```